lte/gateway/python/magma/pipelined/app/enforcement_stats.py

```diff
@@ -703,3 +703,9 @@ def _get_policy_type(match):
     if SCRATCH_REGS[1] not in match:
         return None
     return match[SCRATCH_REGS[1]]
+
+def _get_tunnel_id(flow):
+    if 'tunnel_id' in flow.match:
```

Comment:
lets follow the above pattern
```
def _get_tunnel_id(flow):
    if 'tunnel_id' not in flow.match:
        return None
    return flow.match['tunnel_id']
```

Reply:
and lets handle the None case when calling _get_tunnel_id as an error case

Reply:
Done changes.